invoicedapi/Util/CommonUtil.cs

```diff
@@ -4,7 +4,7 @@ namespace Invoiced
 {
     public static class CommonUtil
     {
-        public static Dictionary<string, string> parseLinks(string links)
+        public static Dictionary<string, string> ParseLinks(string links)
```

Comment:
It's possible that users have depended upon the original function name. Can you please change this back?

Reply:
done